src/main/java/karuna/karuna_backend/visitor/message/domain/VisitorMessageService.java

```diff
@@ -10,9 +11,12 @@
 public class VisitorMessageService {
 
     private final VisitorMessageRepository visitorMessageRepository;
+    private final EmailSender emailSender;
 
     public VisitorMessageDto sendMessage(VisitorMessageCreateDto visitorMessageCreateDto) {
         VisitorMessage visitorMessage = VisitorMessageMapper.toModel(visitorMessageCreateDto);
-        return VisitorMessageMapper.toDto(visitorMessageRepository.save(visitorMessage));
+        VisitorMessage savedVisitorMessage = visitorMessageRepository.save(visitorMessage);
+        emailSender.sendEmail("HARDCODED-EMAIL@EMAIL.PL", "KARUN VISTOR MESSAGE", savedVisitorMessage.getBody(), null);
+        return VisitorMessageMapper.toDto(savedVisitorMessage);
```

Comment:
If I understand the logic correctly and Harcoded-email is private email to which vistiro message should be sent from our smpt server let's say.  I do not see contact email being sent. So we sent only content of the message provided by visitor but we are not sending any kind of way to contact him back am I correct?

Reply:
Yes, we just send email with message. In future we should use template which contain contact and body